docs/Makefile

```diff
@@ -57,9 +57,9 @@ docs-coverage:
 	PYTHONPATH=$(PYTHONPATH) $(SPHINXBUILD) -b coverage $(ALLSPHINXOPTS) $(BUILDDIR)/coverage
 
 html:
-	PYTHONPATH=$(PYTHONPATH) $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	PYTHONPATH=$(PYTHONPATH) $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/$(CURBRANCH)
```

Comment:
Posted this on sdcflows by accident:

> This is going to make it trickier to rerender older versions at need. Or rather, it will create a watershed where we do one thing to render before and another thing to render after. Even if we can't get a single "correct" result now, I would suggest that we leave this rule unchanged and create a new rule like `htmlbranch`.